tracer/test/benchmarks/Benchmarks.Trace/NLogBenchmark.cs

```diff
@@ -56,7 +59,27 @@ public void EnrichedLog()
             {
                 using (LogInjectionTracer.StartActive("Child"))
                 {
+                    object state = null;
+
+                    if (DiagnosticContextHelper.Cache<NLog.Logger>.Mdlc is { } mdlc)
```

Comment:
Should we call `DiagnosticContextHelper.Cache<NLog.Logger>.Mdlc` in the static constructor to avoid the startup code path? Or did you specifically want to test that in the microbenchmark?

Reply:
I think we should do it in the constructor. Though as Benchmarkdotnet always does warm up runs, I think the end result will be the same

Reply:
Added to the constructor in 8e280fbaeb36d290b1788ef949d74e4671193ff6